internal/api/data/repos.go

```diff
@@ -12,6 +12,22 @@ type Repo struct {
 	ProjectDescription string `json:"projectDescription"`
 }
 
+func GetReposDbUrl(ctx context.Context, db *sql.DB, org string, repo string) (string, error) {
+	query := "SELECT db_url FROM repos WHERE organization = ? AND repository = ?"
+
+	row := db.QueryRowContext(ctx, query, org, repo)
+
+	var dbUrl string
+
+	err := row.Scan(&dbUrl)
+
+	if err != nil {
+		return "", err
+	}
+
+	return dbUrl, nil
```

Comment:
What's this for ?

Reply:
https://github.com/dxta-dev/app/blob/6bf793f32620ab00ed64cb4ed77d420bf69ab3c2/internal/api/handler/api.go#L71

Reply:
oh ok, not sure why I don't see a delete of old code, guess it's golang stuff